libs/platform/approval-flow/approval-flow.component.ts

```diff
@@ -1164,6 +1174,24 @@ export class ApprovalFlowComponent implements OnInit, OnChanges, OnDestroy {
             });
         this._subscriptions.add(sub);
     }
+
+    /** @hidden */
+    private removeSpaceNodesFromColumns(columns: ApprovalGraphColumn[]): ApprovalGraphColumn[] {
+        const filteredColumns: ApprovalGraphColumn[] = [];
+
+        columns.forEach((column) => {
+            const filteredNodes = column.nodes.filter((node) => !node.space);
```

Comment:
LGTM, but I believe `space` is there for reason. Not sure if removing it triggers side effects. It would be great if this gets reviewed by fundamental team.